plugin/trino-hive/src/main/java/io/trino/plugin/hive/security/AccessControlMetadata.java

```diff
@@ -34,6 +34,11 @@ default boolean isUsingSystemSecurity()
         return false;
     }
 
+    default boolean supportsRole()
+    {
+        return false;
```

Comment:
"supportsRoles" (plural)?

also, this should be used in default implementations of all other role-related methods.